src/swarm/neo/request/RequestEventDispatcher.d

```diff
@@ -823,8 +826,8 @@ public struct RequestEventDispatcher
     {
         while ( this.queued_signals.length > 0 )
         {
-            auto code = this.queued_signals[0];
-            Array.removeShift(this.queued_signals, 0);
+            auto code = this.queued_signals.array()[0];
+            .removeShift(this.queued_signals, 0);
```

Comment:
If it's just this one location, it might be better to just use `memmove` directly, rather than implementing a generic `removeShift` function.

Reply:
Fair enough.